engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/DebuggingEnsoTest.java

```diff
@@ -317,6 +317,41 @@ public void testEvaluateExpression() {
     }
   }
 
+  @Test
+  public void testEvaluateExpressionInDebugBreakpoint() {
+    Value fooFunc =
+        createEnsoMethod(
+            """
+
+        import Standard.Base.Runtime.Debug
+
+        foo x =
+            a = 6
+            b = 7
+            Debug.breakpoint
+        """,
+            "foo");
+
+    int[] res = {0};
+    try (DebuggerSession session =
+        debugger.startSession(
+            (SuspendedEvent event) -> {
+              switch (event.getSourceSection().getCharacters().toString().strip()) {
+                case "Debug.breakpoint" -> {
+                  DebugStackFrame stackFrame = event.getTopStackFrame();
+                  DebugValue evaluatedValue = stackFrame.eval("a * b");
+                  assertTrue(evaluatedValue.isNumber());
+                  assertEquals(42, res[0] = evaluatedValue.asInt());
```

Comment:
Operator `=` returns value? I had no idea, this looks like C :D. But if it works ...

Reply:
Old good C-like syntax.